docs/cookbook/prompt_engineering/chaining_based/self_verification.md

```diff
@@ -76,9 +75,11 @@ async def mask_prompt(query: str) -> openai.OpenAIDynamicConfig:
 @prompt_template(
     """
     SYSTEM:
-    You will be given a masked prompt with some value Xd out, and the solution to
-    the original problem. Return the prompt verbatim, but fill in the value for X
-    according to what you see in the solution.
+    You will be given a masked prompt with some value Xd out, and the solution to \
```

Comment:
this won't really work because it will introduce a tab into the prompt.

It's ok for these single sentences to expand beyond the ruler limit